client/src/pages/courses/view.tsx

```diff
@@ -0,0 +1,150 @@
+import Checkbox from '@/src/shared/components/Checkbox';
+import Dropdown from '@/src/shared/components/Dropdown';
+import Radiobutton from '@/src/shared/components/Radiobutton';
+import SearchBar from '@/src/shared/components/SearchBar/SearchBar';
+import GridIcon from '@/src/shared/icons/GridIcon';
+import ListIcon from '@/src/shared/icons/ListIcon';
+import { LeftSideBar } from '@/src/shared/layouts/RightSideBar/LeftSideContainer';
+import { RightSideBar } from '@/src/shared/layouts/RightSideBar/RightSideBar';
+import Image from 'next/image';
+import { Fragment, useState } from 'react';
+import type { ReactNode } from 'react';
+
+const View = (): ReactNode => {
```

Comment:
- i noticed most of the code lacks an indention or has improper tab spaces. 
- format all documents